sklearn/feature_extraction/text.py

```diff
@@ -1474,13 +1479,25 @@ class TfidfVectorizer(CountVectorizer):
 
     See also
     --------
-    CountVectorizer
-        Tokenize the documents and count the occurrences of token and return
-        them as a sparse matrix
-
-    TfidfTransformer
-        Apply Term Frequency Inverse Document Frequency normalization to a
-        sparse matrix of occurrence counts.
+    CountVectorizer : Produces a sparse representation of the counts using
```

Comment:
In the context of TfidfVectorizer this is not explaining the difference well. Just "Transforms text into a sparse matrix of n-gram counts" is sufficient. So is the current wording.